packages/fxa-event-broker/src/queueworker/sqs.dto.ts

```diff
@@ -91,10 +91,48 @@ export const PROFILE_CHANGE_SCHEMA = joi
   .unknown(true)
   .required();
 
-export type deleteSchema = joi.Literal<typeof DELETE_SCHEMA>;
-export type loginSchema = joi.Literal<typeof LOGIN_SCHEMA>;
-export type passwordSchema = joi.Literal<typeof PASSWORD_CHANGE_SCHEMA>;
-export type profileSchema = joi.Literal<typeof PROFILE_CHANGE_SCHEMA>;
-export type subscriptionUpdateSchema = joi.Literal<
-  typeof SUBSCRIPTION_UPDATE_SCHEMA
->;
+export type deleteSchema = {
+  event: 'delete';
```

Comment:
I tried `event: DELETE_EVENT`, but got the following errors 🤨:
<img width="901" alt="Screen Shot 2022-05-31 at 3 43 22 PM" src="https://user-images.githubusercontent.com/28129806/171271924-2326ed62-8ce9-454a-93c9-bcf9cafdeada.png">

Reply:
Yep, the error message does mention how to fix this. I believe the typeof DELETE_EVENT will be a concrete string as desired.

Reply:
Hm, I thought since it was `event: joi.string().valid(DELETE_EVENT).required()` that `event: 'delete'` would be its equivalent, as `typeof DELETE_EVENT` would just be `string`?

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-8.html#string-literal-types

Reply:
Noting that `typeof DELETE_EVENT` maintains the narrow type and, when used, a string other than `delete` would not be valid. 

Source: Ben's test :) 
https://www.typescriptlang.org/play?#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCbvCwDKgU8JkY7p7ehCTkVDQS2E6gnPCxGcwmZqDSTgzxxWWVoASMFmgYkAAeRJTInN3ymj4d-jSCeNsMq-wuoPaOltigAKoASgAywhK7SbGQZIIz5VWCFzSeCrZagNYbChbHaxUDcCjJZLfSDbExIAgUdxkUBIursJzCFJtXydajBBCcQQ0AAiAFFnrScLSAPq0gBqtIAcjhQABeUAAIm4MEgCgFilQ7TQ+lQAEY+aAAN6KEolSB6bb5UAAcmF7gU2sUAF8JVKFFSAEwK5Wq8Ea1BayVEIl0hlM1kc7nG4LJLiweqNUDsQSQZ7yb4MaAACng0gAVo6ZbKAJRKlWqimCeDuALQeDMKPazjqSDa5PexS+8wBrzByDgBwKPRkGPxxOW1M2jONLM5vMF7WYRCIMvpkp1sMKMiR1tx8smxSZ7OQXP5qN1ht1dXfKOK7eanV60Wlo3J8vBFQ4ZoeMheRDwPoMYHsGhfNxmZjsBjMSD8WDQdREifeA9BELwBVpVYGEQBZ9AFJJsQFTdUG1YROUgCQBSCFQAHkvjId4Q34H8jlQRwFFcaQ8FAJx7x0YYsH0FAEGHFFUGES43wkB9ohEAwOGxBYmFYDgeCCEogA